src/views/CreatePage.vue

```diff
@@ -362,4 +393,64 @@ div(class="m-auto border-4 border-orange-100 max-w-screen-xl my-4")
         v-transformer(ref="transformer" :config="configShapeTransformer")
 div(class="container")
   ToolBar(:stage="stage" :stage-parent-div="stageParentDiv" :save-canvas="saveCanvas")
```

Comment:
```suggestion
  ToolBar(:stage="stage")
```
Toolbar.vueにpropsで渡さなくなったので削除